speechbrain/__init__.py

```diff
@@ -34,6 +46,9 @@ def make_deprecated_redirections():
         also_lazy_export=True,
     )
 
+    for old_path, new_path in deprecations.items():
+        deprecated_redirect(old_path, new_path, also_lazy_export=True)
```

Comment:
should we also add an `extra_reason` ?

Reply:
Sure, this can be added.

Reply:
This is the message, do you think there's something else we should add here? Like version or something?

```
>>> from speechbrain.wordemb import transformer
<stdin>:1: UserWarning: Module 'speechbrain.wordemb' was deprecated, redirecting to 'speechbrain.integrations.huggingface.wordemb'. Please update your script.
```